std_msgs/Header header
std_msgs/Float64 range_min
std_msgs/Float64 range_max
